sources/api/schnauzer/src/helpers/mod.rs

```diff
@@ -1314,6 +1320,20 @@ fn oci_spec_resource_limits(
     Ok(serde_json::from_value(value.clone())?)
 }
 
+/// This helper adjusts resource limits for the case when EFA attachment is detected.
+/// We need to increase the locked memory limits from the default 8096KB to unlimited
+/// to account for hugepages allocation.
+fn adjust_resource_limits_for_efa(
+    rlimits: &mut HashMap<OciDefaultsResourceLimitType, OciDefaultsResourceLimitV1>,
+) {
+    rlimits
+        .entry(OciDefaultsResourceLimitType::MaxLockedMemory)
+        .or_insert(OciDefaultsResourceLimitV1 {
+            soft_limit: -1,
+            hard_limit: -1,
```

Comment:
Should a constant be created as `UNLIMITED = -1`? The `-1` doesn't mean too much if we don't provide context somehow, we are just assuming that `-1` means unlimited.